src/index.js

```diff
@@ -106,19 +120,21 @@ async function fetchWithNetworkEvents(el: Element, url: string, options: Request
 }
 
 async function check(autoCheckElement: AutoCheckElement) {
-  const src = autoCheckElement.src
-  if (!src) {
-    throw new Error('missing src')
```

Comment:
Why are these now changed to returns?

Reply:
I changed this to returns so that I could group all of the early exit conditions in one `if` statement where I can clear the custom validity once.